app/lib/constants/site_config.rb

```diff
@@ -2,11 +2,15 @@ module Constants
   module SiteConfig
     DETAILS = {
       allow_email_password_registration: {
-        description: "Can users sign up with only email and password?",
+        description: "Can users sign up using their email and password?",
```

Comment:
I changed the language of the `description` here to something that makes more sense to me (and perhaps to @vaidehijoshi, as she mentioned being a little confused with the word `only` 😄 )